docker-compose.yml

```diff
@@ -24,129 +24,7 @@ services:
       - sdk-harness
     volumes:
       - genesis-file:/genesis-file
```

Comment:
Removing all indexer dockers for now. We should bring them back in when we have a live test that uses them.